selfdrive/car/toyota/carcontroller.py

```diff
@@ -32,9 +33,11 @@ def __init__(self, dbc_name, CP, VM):
     self.alert_active = False
     self.last_standstill = False
     self.standstill_req = False
-
     self.steer_rate_limited = False
 
+    self.use_interceptor = CP.enableGasInterceptor
```

Comment:
Why not init to False?

Reply:
That's fair, I forget most cars openpilot can start while driving. I was assuming we always starts from 0